lib/runtime.d.ts

```diff
@@ -10,6 +10,11 @@ interface WebpackRequire {
     (id: string): any;
     (paths: string[], callback: (...modules: any[]) => void): void;
     ensure(ids: string[], callback: WebpackRequireEnsureCallback, chunkName?: string): void;
+    context(directory: string, useSubDirectories: boolean, regExp: RegExp): WebpackContext;
```

Comment:
Thanks!! Could you please make `useSubDirectories` and `regExp` optional? http://webpack.github.io/docs/context.html